SitumWayfinding/Classes/Positioning/PositioningViewController.swift

```diff
@@ -291,11 +291,11 @@ class PositioningViewController: UIViewController ,GMSMapViewDelegate, UITableVi
     //MARK: Floorplans
     
     func displayMap(forLevel selectedLevelIndex: Int) {
-        let levelIdentifier = buildingInfo!.floors[selectedLevelIndex].identifier
+        let levelIdentifier = orderedFloors(buildingInfo: buildingInfo!)[selectedLevelIndex].identifier
```

Comment:
Creo que sería mejor que el parámetro de orderedFloors fuese un optional, para no tener que forzar el unwrapeado de buildingInfo sin comprobar si es nil. Si por alguna razón esa buildingInfo fuese nil, provocaría un error
`Trying to use ! to access a nonexistent optional value triggers a runtime error. Always make sure that an optional contains a non-nil value before using ! to force-unwrap its value.`
En este caso habría que usar optional bindind o optional chaining dentro de la función